lib/connect-backbone-to-react.js

```diff
@@ -119,7 +121,8 @@ module.exports = function connectBackboneToReact(
         // The only case where this flag is encountered is when this component
         // is unmounted within an event handler but the 'all' event is still triggered.
         // It is covered in a test case.
-        if (this.hasBeenUnmounted) {
+        // Also bails if we haven't yet mounted, to avoid warnings in strict mode.
+        if (!this.componentIsMounted) {
```

Comment:
No way to test this, is there?

Reply:
Maybe a general smoke test, rendering in strict mode and make sure `console.error` isn't called? Other than that, nothing comes to mind...